packages/matchbox/src/components/ProgressBar/ProgressBar.js

```diff
@@ -22,25 +23,28 @@ export const StyledProgressBarInner = styled(Box)`
   ${calculatedWidth}
 `;
 
-function ProgressBar(props) {
+const ProgressBar = React.forwardRef(function ProgressBar(props, userRef) {
   const { completed = 0, label, size, valueText, ...rest } = props;
+  const systemProps = pick(rest, system.propNames);
 
   return (
     <StyledProgressBarOuter
       as="div"
       visualSize={size}
-      {...rest}
       role="progressbar"
       aria-label={label}
       aria-valuenow={`${completed}`}
       aria-valuemin="0"
       aria-valuemax="100"
       aria-valuetext={valueText}
+      {...systemProps}
+      data-id={rest['data-id']}
```

Comment:
Any reason to do this over just explicitly defining the `data-id` prop in the spread?

Reply:
Oh yea forgot you could do that